### PR TITLE
@orta => [Inquiries] Add response data to inquiry failure analytics

### PR DESCRIPTION
**Pull request by jorystiefel**
@orta please review and make sure this looks good.

### CODE REVIEW

**Comment by orta** [comment]
Giphy changeling

**Comment by jorystiefel** [comment]
Thanks OS X 10.11 autocorrect.

File: Artsy/App/ARAppDelegate+Analytics.m [hunk 1]
```diff
@@ -412,6 +412,7 @@ - (void)setupAnalytics
                             ARAnalyticsProperties: ^NSDictionary*(ARInquireForArtworkViewController *controller, NSArray *parameters) {
                                 return @{
                                     @"errors" : [[parameters first] localizedDescription] ?: @"",
+                                    @"response" : [[parameters objectAtIndex:1] localizedDescription] ?: @"",
```
Comment (orta) [inline Artsy/App/ARAppDelegate+Analytics.m]: `NSError *error, NSHTTPURLResponse *response` - the second is the response object, you probably want the content or body

Comment (orta) [inline Artsy/App/ARAppDelegate+Analytics.m]: Actually, I think you can get the data from the error itself!
See here: https://github.com/artsy/eigen/blob/master/Artsy/Networking/ArtsyAPI.m#L77-L90

```
        NSData *data = error.userInfo[AFNetworkingOperationFailingURLResponseDataErrorKey];
        NSDictionary *recoverySuggestion = [NSJSONSerialization JSONObjectWithData:data options:0 error:nil];
```

Comment (jorystiefel) [inline Artsy/App/ARAppDelegate+Analytics.m]: Actually, looking at [the docs](https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSHTTPURLResponse_Class/#//apple_ref/occ/clm/NSHTTPURLResponse/localizedStringForStatusCode:) I wonder if the `NSHTTPURLResponse` is what we want at all. This only returns the HTTP response _headers_, which doesn't really help us, does it?

Comment (jorystiefel) [inline Artsy/App/ARAppDelegate+Analytics.m]: Will investigate!

Comment (orta) [inline Artsy/App/ARAppDelegate+Analytics.m]: Yeah, I agree

**Comment by jorystiefel** [comment]
@orta I think this is what we want now.

**Comment by orta** [comment]
Yeah, me too :+1:

**Comment by alloy** [comment]
:+1:

**Pull request merged**